cg/cli/clean.py

```diff
@@ -277,6 +278,15 @@ def clean_flow_cells(context: CGConfig, dry_run: bool):
             continue
 
 
+@clean.command("retrieved-spring-files")
+@DRY_RUN
+@click.pass_obj
+def clean_retrieved_spring_files(context: CGConfig, dry_run: bool):
+    """Command for cleaning Spring files which were retrieved more than 7 days ago."""
+    clean_retrieved_spring_files_api = CleanRetrievedSpringFilesAPI(context.housekeeper_api)
+    clean_retrieved_spring_files_api.clean_retrieved_spring_files(dry_run)
```

Comment:
I would like to make the dry_run an attribute of the API class, to avoid being passed around through the API's methods, but that is my taste, what do you think? You could pass it directly to the constructor or make a method `set_dry_run`

Reply:
Implemented it!